doc/source/features.rst

```diff
@@ -80,22 +80,16 @@ Command line usage will use the new names ::
       {friendly_func,func2}
 
 Nested Subcommands
-------------------
```

Comment:
keep underlining as before.

Reply:
Ok, I thought you may want it a sub section under the subcommands.  I will promote it.